fe/fe-core/src/test/java/com/starrocks/qe/ConnectSchedulerTest.java

```diff
@@ -68,26 +64,6 @@ public void setUp() throws Exception {
 
     @Test
     public void testSubmit(@Mocked ConnectProcessor processor) throws Exception {
-        // mock new processor
```

Comment:
Why these are useless?

Reply:
To be honest, I don't know, but I'm sure these cases haven't checked the results of processor work